public/styles/sitestyle.css

```
@@ -349,3 +349,8 @@ body{
 #gifApp{
 	margin: 10px 0 0;
 }
+
+.inputControl-addon{
+	margin-right: 30px
```

There is a semicolon missing here.

```suggestion
	margin-right: 30px;
```